build.rs

```diff
@@ -0,0 +1,24 @@
+#[path = "src/args.rs"]
+mod args;
```

Comment:
```suggestion
#[allow(dead_code)]
mod args;
```
seems to fix the warning